examples/SampleApp/src/components/MessageSearch/MessageSearchList.tsx

```diff
@@ -81,6 +82,7 @@ export const MessageSearchList: React.FC<MessageSearchListProps> = ({
   refreshing,
   refreshList,
   showResultCount = false,
+  scrollRef,
```

Comment:
Lets use `React.forwardRef` instead of passing ref as a prop.

Reply:
Resolved